src/screens/Messages/List/ChatListItem.tsx

```diff
@@ -152,102 +144,119 @@ function ChatListItemReady({
               ]
             : undefined
         }
-        onAccessibilityAction={onLongPress}>
+        onAccessibilityAction={onLongPress}
+        onPress={
+          isDeletedAccount
+            ? e => {
+                e.preventDefault()
+                return false
+              }
+            : undefined
+        }
```

Comment:
Cool for now, but we should make this a prop on our new link maybe, I've had to do this a few times before. Just something for later.